po/fi.po

```diff
@@ -1,18 +1,19 @@
-# gnucash/fi.
+# Finnish translation of GnuCash.
 # This file is distributed under the same license as the gnucash package.
 # Copyright (C) 2008 gnucash project
+# Marko Kohtala <marko.kohtala@gmail.com>, 2020.
 # Tuomo Hartikainen <tth@harski.org>, 2017.
 # Tuomo Kohvakka <tuomo.kohvakka@iki.fi>, 2008.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: GnuCash 2.6.15\n"
+"Project-Id-Version: GnuCash 3.10\n"
 "Report-Msgid-Bugs-To: https://bugs.gnucash.org/enter_bug.cgi?"
 "product=GnuCash&component=Translations\n"
 "POT-Creation-Date: 2020-03-28 12:48-0700\n"
 "PO-Revision-Date: 2017-03-13 20:59+0200\n"
-"Last-Translator: Tuomo Hartikainen <tth@harski.org>\n"
```

Comment:
Had you talked with Tuomo?

Reply:
I had contacted @harski in 2018. He told he got some other work and was happy someone would be continuing. I understood he would not mind. I just tonight remembered to drop him a note I've created this pull request in case he has time to take a look.

Reply:
I'm cool with this, I haven't done any further translations since we last spoke.